bot/exts/info/pypi.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from random import choice
```

Comment:
Can we directly just do `import random`. Just looking at `choice` In the code is sometimes confusing and adding `random.choice()` would be much more better

Reply:
Out of context, yes I can see how that's confusing. Will be changed.

Reply:
Decided to go with `itertools.cycle` so it can rotate through 3 colors infinitely. Those being yellow, blue, and white.